src/modules/Snapper.rb

```diff
@@ -23,14 +23,119 @@
 # Summary:	Snapper settings, input and output functions
 # Authors:	Jiri Suchomel <jsuchome@suse.cz>
 #
-# $Id$
-#
 # Representation of the configuration of snapper.
 # Input and output routines.
+
 require "yast"
 
 module Yast
+
+
+  class Tree
```

Comment:
@aschnell Yes, I had a similar issue with `yast-storage`. Anyway, in this case just rename the class to `SnapperTree` will do the trick. Anyway, leave it as is if you prefer.